src/app/dashboard/executionsgrid/gridrow/gridrow.component.html

```diff
@@ -0,0 +1,99 @@
+<ng-template #childComponentTemplate>
+  <ng-template #empty_date><span class="time">--</span></ng-template>
+
+  <span
+    [ngClass]="{ active: expanded }"
+    class="grid-cell row-start"
+    (click)="toggleExpand($event)"
+    title="{{ dsExecution.dataset.datasetName }}"
+  >
+    {{ dsExecution.dataset.datasetName }}
+    <span class="br">
+      <a [routerLink]="['/dataset', 'edit', dsExecution.dataset.datasetId]" target="_blank">{{
+        dsExecution.dataset.datasetId
+      }}</a>
+    </span>
+  </span>
+
+  <span class="grid-cell row-steps" *ngIf="dsExecution.executionProgress">
+    {{ dsExecution.executionProgress.stepsDone }} / {{ dsExecution.executionProgress.stepsTotal }}
+    {{ 'overviewsteps' | translate }}
+    <ul
+      *ngIf="dsExecution.execution.plugins && dsExecution.execution.plugins.length > 0"
+      class="orbs"
+    >
+      <li
+        *ngFor="let pi of dsExecution.execution.plugins"
+        class="{{ pi.pluginType }} orb-status {{ getPluginStatusClass(pi) }}"
+      ></li>
+    </ul>
+  </span>
+
+  <span class="grid-cell" *ngIf="dsExecution.executionProgress">
+    <span *ngIf="dsExecution.executionProgress.currentPluginProgress">
```

Comment:
Is there a way to make this shorter? I mean, "dsExecution.executionProgress.currentPluginProgress" is repeated a couple of times here.